src/test/java/uk/gov/hmcts/divorce/citizen/notification/Applicant2NotBrokenNotificationTest.java

```diff
@@ -151,4 +151,27 @@ void shouldSendEmailToApplicant2WithDissolutionContent() {
         );
         verify(commonContent).mainTemplateVars(data, 1234567890123456L, data.getApplicant2(), data.getApplicant1());
     }
+
+    @Test
+    void shouldSendWelshEmailToApplicant2WhenWelshLanguagePreference() {
+        CaseData data = validJointApplicant1CaseData();
+        data.getApplicant2().setLanguagePreferenceWelsh(YesOrNo.YES);
+
+        final Map<String, String> templateVars = Map.of(IS_DIVORCE, YES, IS_DISSOLUTION, NO);
+        when(commonContent.mainTemplateVars(data, 1234567890123456L, data.getApplicant2(), data.getApplicant1()))
+            .thenReturn(templateVars);
+
+        notification.sendToApplicant2(data, 1234567890123456L);
```

Comment:
Tiny suggestion - Could we use the TEST_CASE_ID variable instead of the string in place